packages/lesswrong/components/questions/NewRelatedQuestionForm.jsx

```diff
@@ -0,0 +1,73 @@
+import { Components, registerComponent, getFragment, withMessages } from 'meteor/vulcan:core';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import withUser from '../common/withUser'
+import { Posts } from '../../lib/collections/posts/collection.js'
+
+const styles = theme => ({
+  answersForm: {
+    maxWidth:650,
+    paddingBottom: theme.spacing.unit*4,
+    [theme.breakpoints.down('md')]: {
+      marginLeft: "auto",
+      marginRight: "auto"
+    }
+  },
+  formButton: {
+    paddingBottom: "2px",
+    fontSize: "16px",
+    marginLeft: "5px",
+    "&:hover": {
+      background: "rgba(0,0,0, 0.05)",
+    },
+    color: theme.palette.secondary.main,
+    float: "right"
+  },
+  formSubmit: {
+    display: "flex",
+    flexWrap: "wrap",
+  }
+})
+
+const NewRelatedQuestionForm = (props) => {
+  const { post, classes, flash, currentUser, refetch } = props
+  const { SubmitToFrontpageCheckbox, PostSubmit } = Components
+
+  const QuestionSubmit = (props) => {
+    return <div className={classes.formSubmit}>
+      <SubmitToFrontpageCheckbox {...props}/>
+      <PostSubmit {...props} />
+    </div>
+  }
+  if (!currentUser) return null
```

Comment:
flagging that this currently behaves different from other comment boxes after Jim's update. How much work is it to apply jim's update here?